float64 distance
Header header
geometry_msgs/Pose distance_vector
geometry_msgs/Twist frame_twist
geometry_msgs/Vector3 collision_pnt_vector